main.go

```diff
@@ -186,22 +170,7 @@ func main() {
 	}
 
 	// Initialize Kafka Producer
-	kafkaConfig := kafka.ConfigMap{
-		"bootstrap.servers": config.Ingest.Kafka.Broker,
-	}
-	if config.Ingest.Kafka.SecurityProtocol != "" {
-		kafkaConfig["security.protocol"] = config.Ingest.Kafka.SecurityProtocol
-	}
-	if config.Ingest.Kafka.SecurityProtocol != "" {
-		kafkaConfig["sasl.mechanism"] = config.Ingest.Kafka.SaslMechanisms
-	}
-	if config.Ingest.Kafka.SecurityProtocol != "" {
-		kafkaConfig["sasl.username"] = config.Ingest.Kafka.SaslUsername
-	}
-	if config.Ingest.Kafka.SecurityProtocol != "" {
-		kafkaConfig["sasl.password"] = config.Ingest.Kafka.SaslPassword
-	}
```

Comment:
@hekike this looks like a copy-paste error. Can you please confirm if it is?

Reply:
ouch, yes it is

Reply:
do you plan to fix it in this PR or should I open a fix?

Reply:
Yes, I'll take care of it.